gnucash/import-export/log-replay/gnc-log-replay.h

```diff
@@ -29,7 +29,7 @@
 
 /** The gnc_file_log_replay() routine will pop up a standard file
  *     selection dialogue asking the user to pick a log file to replay. If one
- *     is selected the the .log file is opened and read.  It's contents
+ *     is selected the .log file is opened and read.  Its contents
```

Comment:
No, AFAIK in English genitive is `~'s`,
plural often `~s` and
genitive plural then `~s'`.
@jralls right?

Reply:
Nope, because of the collision with the contraction of "it is" that takes precedence over the possessive. "It's" is the contraction, "its" is the genetive/possessive. It's a common error, even native speakers screw it up about as often as they use "loose" (release from confinement or make not tight) when they want "lose" (give up possession or misplace).